rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/directives/AWSProtocolDirectives.scala

```diff
@@ -0,0 +1,23 @@
+package org.elasticmq.rest.sqs.directives
+
+import akka.http.scaladsl.model.{ContentType, MediaType}
+import akka.http.scaladsl.server.Directives.extractRequest
+import org.elasticmq.rest.sqs.AWSProtocol
+
+trait AWSProtocolDirectives {
+
+  import AWSProtocolDirectives._
+  def extractProtocol = extractRequest.map { request =>
+    request.entity.contentType match {
```

Comment:
I'm not sure about this. Should we already support another version of protocol? I know there's already 1.1, but I'm not sure what is the difference.